README.md

```
@@ -1,135 +1,38 @@
-# Turborepo starter
+#  शॉर्ट-डोमेन-मोनो
```

markdownlint-cli2 (0.18.1)

Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)